apps/challenges/aws_utils.py

```diff
@@ -1067,6 +1067,38 @@ def get_logs_from_cloudwatch(
     return logs
 
 
+def get_task_status(task_def_arn):
+    """
+    To fetch current status of worker for the given task_def_arn
+    """
+    client = get_boto3_client("ecs", aws_keys)
+    worker_status = {}
+    if settings.DEBUG:
+        worker_status = {
+            "error": False,
+            "details": "Not available for development environment"
+        }
+    else:
+        try:
+            response = client.describe_tasks(
+                tasks=[
+                    task_def_arn
```

Comment:
@burnerlee this API won't work directly for us. We store task definition for each service in our challenge model. To get the list of tasks for which we want the status you have to first store `serviceName` in our model, this can be done while creating the fargate service for each challenge. You can look at boto3 API call in `start_workers` and make required changes to store service name in our models. 

Once you have the service name you can use `client.list_tasks` to list the tasks that are active in  the service and fetch status for each task from that response using `client.describe_task` API.